### ISSUE TITLE
add header description in the fasta file used for the mixed species modules.

### ISSUE THREAD

**Issue by mlocardpaulet** [Proteobench/ProteoBench#463]
Some software tools need the header description to properly parse fasta header with default parameters. We do not have this in the current fasta proposed for the DDA and DIA modules.

So we decided to generate a fasta with these descriptions and the same sequences as the ones in the [fasta](https://proteobench.cubimed.rub.de/datasets/fasta/ProteoBenchFASTA_Quantification.zip) that we currently use.

I had a look at it and: problem!

Whatever I do, we won't have description for all the accessions because:
1. some accessions are "weird" but necessary: the spiked in biognosys iRTs have no descriptions; and some sequences in the contaminants are tags or else (hence, no header description).
2. some accessions are deleted or not annotated anymore in Uniprot.

So, whatever I do there will be accessions without descriptions. 

Will this be an issue for some software tools? If yes, how do I proceed? What do I do with the sequences that are not "real" proteins? 

@wolski: do you have an opinion on how to proceed?

**Comment by wolski** [comment]
Hi @mlocardpaulet 

I used the UniProt service to retrieve the description lines for all the UniProt ids,  for all the entries in
[ProteoBenchFASTA_DDAQuantification.zip](https://proteobench.cubimed.rub.de/datasets/fasta/ProteoBenchFASTA_Quantification.zip)

I am attaching the jupyter notebook.
[AddDescriptions.ipynb.zip](https://github.com/user-attachments/files/18605569/AddDescriptions.ipynb.zip)

I created a new fasta with the description lines. I uploaded the file to the proteobench cloud.
It is called:
ProteoBenchFASTA_DDAQuantification_Descriptions.fasta
and you will find it in the Module_2_DDA... folder. 

Compared with the original fasts, the only change is that all entries except about 40 now have a description.

**Comment by wolski** [comment]
@mlocardpaulet, you are correct; we can't be sure if adding the description line does not change things, so we must stay on the original fasta. Let's close the ticket.

**Comment by mlocardpaulet** [comment]
Sounds good. Thanks for looking into it!
